### DISCUSSION TITLE
Changing everything to APA7

### DISCUSSION THREAD

**Discussion by db-leonardo** [joaomlourenco/novathesis#508] [Q&A] [answered]
Hello, thank you for this template that helped me throughout this phase!

Now that I have to submit my dissertation, I need to solve a problem: I need everything to be according to APA7. I thought it was enough to cite everything using APA, but I also need figures and tables to be according to APA7.
Here are the examples:
![WhatsApp Image 2024-07-22 at 19 34 10_c4293d4d](https://github.com/user-attachments/assets/902486ab-373c-49fb-999c-15cdf51f3b00)
![image](https://github.com/user-attachments/assets/e93b3d64-f46e-46e8-9025-6a8178d53a5a)

I've seen that there is a class "apa7" that could do the work, but how could I change the class in novathesis?

I am using VS Code and Tex Live.

Thanks in advance!

**Comment by joaomlourenco** [comment]
@db-leonardo just to make it clear.
What are your concrete needs?
1) The numbering of the table/figure in Bold and on a line by itself ("Figura 4")
2) The caption in italics below the figure/table number
3) Both for tables and figures, the number and caption are above the table/figure

Are these all?  Or something else?

**Reply by joaomlourenco** [reply]
@db-leonardo just because I am curious, what is your University/School/Degree?  Thanks.

**Reply by db-leonardo** [reply]
Yes, is that what I need.
I'm from FTC-NOVA.

**Comment by joaomlourenco** [comment, marked as answer]
Add the following to the file `Config/5_packages.tex`

```latex
\usepackage[
	labelfont=bf,
	justification=RaggedRight,
	singlelinecheck=off,
	textfont=it,
	labelseparator=newline,
]{caption}
```
and let me know if it works for you.

_**Disclaimer:** The template NOVAthesis and this user support **are not part of my job description** at NOVA FCT, thus they take on my resting and family time.  So, if this was helpful, I would appreciate if you visit my “[tip jar](https://www.paypal.com/donate/?hosted_button_id=8WA8FRVMB78W8)”_.

**Reply by db-leonardo** [reply]
That helped, thank you!
